src/frontend/reducers.js

```diff
@@ -15,4 +15,4 @@ export default req.keys().reduce((acc, key) => {
   acc[name] = req(key);
 
   return acc;
-}, { router: routerStateReducer, form: formReducer });
+}, { routing: routeReducer, form: formReducer });
```

Comment:
Some people (me) depends on `router` key in private codebases.
Could you add migration notice to README?